src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -509,40 +510,63 @@ public String getAudioCodec() {
 	@Override
 	public String toString() {
 		StringBuilder result = new StringBuilder();
-		result.append("id: ");
-		result.append(getId());
-		result.append(", lang: ");
-		result.append(getLang());
+		if (!getLang().equals("und")) {
+			result.append("Id: ").append(getId());
+			result.append(", Language Code: ").append(getLang());
+		}
 
 		if (isNotBlank(getAudioTrackTitleFromMetadata())) {
-			result.append(", audio track title from metadata: ");
-			result.append(getAudioTrackTitleFromMetadata());
+			if (result.length() > 0) {
+				result.append(", ");
+			}
+			result.append("Audio Track Title From Metadata: ").append(getAudioTrackTitleFromMetadata());
 		}
 
-		result.append(", audio codec: ");
-		result.append(getAudioCodec());
-		result.append(", sample frequency:");
-		result.append(getSampleFrequency());
-
-		if (getAudioProperties() != null && getAudioProperties().getNumberOfChannels() != 0) {
-			result.append(", number of channels: ");
-			result.append(getAudioProperties().getNumberOfChannels());
+		if (result.length() > 0) {
+			result.append(", ");
+		}
+		result.append("Audio Codec: ").append(getAudioCodec());
+		if (getSampleFrequency() != null) {
+			result.append(", Sample Frequency: ").append(getSampleFrequency());
+		}
+		if (getBitRate() != 0) {
+			result.append(", Bitrate: ").append(getBitRate());
+		}
+		if (getBitsperSample() != 0 && getBitsperSample() != 16 ) {
+			result.append(", Bits per Sample: ").append(getBitsperSample());
 		}
 
-		result.append(", bits per sample: ");
-		result.append(getBitsperSample());
+		if (getAudioProperties() != null) {
+			result.append(", ").append(getAudioProperties());
+		}
+		if (nrAudioChannels != 0) {
+			result.append(", Deprecated Channels: ").append(nrAudioChannels);
+		}
+		if (delay != 0) {
+			result.append(", Deprecated Delay: ").append(delay);
+		}
 
 		if (isNotBlank(getArtist())) {
-			result.append(", artist: ");
-			result.append(getArtist());
-			result.append(", album: ");
-			result.append(getAlbum());
-			result.append(", song name: ");
-			result.append(getSongname());
-			result.append(", year: ");
-			result.append(getYear());
-			result.append(", track: ");
-			result.append(getTrack());
+			result.append(", Artist: ").append(getArtist());
+			if (isNotBlank(getAlbum())) {
+				result.append(", Album: ").append(getAlbum());
+			}
+			if (isNotBlank(getSongname())) {
+				result.append(", Track Name: ").append(getSongname());
+			}
+			if (getYear() != 0) {
+				result.append(", Year: ").append(getYear());
+			}
+			if (getTrack() != 0) {
+				result.append(", Track: ").append(getTrack());
+			}
+			if (StringUtils.isNotBlank(getGenre())) {
+				result.append(", Genre: ").append(getGenre());
+			}
+		}
+
+		if (StringUtils.isNotBlank(getMuxingModeAudio())) {
```

Comment:
@Sami32 Nothing. ```isNotBlank()``` isn't a ```DLNAMediaAudio``` (or parents) method, so for that to be a valid method, you need to import it staticly like this:
```
import static org.apache.commons.lang3.StringUtils.isNotBlank;
```
This is done in this case, but I never use that myself so I didn't think of it. The only reason to do this import is so that you can write ```isNotBlank()``` instead of ```StringUtils.isNotBlank()```, so I simply usually qualify them with ```<class>.<method>``` instead of doing static imports. 

If you use methods from a particular class a LOT in a given code, I guess static import makes sense. In short, in this case, the difference is none 😉